helpValues.go

```diff
@@ -81,17 +96,26 @@ func (h *Help) ExtractValues(p *Parser, message string) {
 			Description:  pos.Description,
 			Required:     pos.Required,
 			DefaultValue: pos.defaultValue,
+			Spacer:       makeSpacer(pos.Name, maxLength),
 		}
 		h.Positionals = append(h.Positionals, newHelpPositional)
 	}
 
+	maxLength = len(versionFlagLongName)
+	if len(helpFlagLongName) > maxLength {
+		maxLength = len(helpFlagLongName)
+	}
+	maxLength = getLongestNameLength(p.subcommandContext.Flags, maxLength)
+	maxLength = getLongestNameLength(p.Flags, maxLength)
```

Comment:
Yes, but intentionally. It also takes the `maxLength` as the 2nd argument which is used as the minimum length. The returned number is the longest of all the flags names but at least the length from line 108. In 108 also the value from line 104 is used as min.

So, what's happening there is, we want to get the max length of all values that are shown together in the [bottom block](https://github.com/integrii/flaggy/pull/50/files#diff-04c6e90faac2675aa89e2176d2eec7d8R66-R71). That is:

- the string `version`. Yeah, it could be hardoced to `7`, but then you would need a comment to explain why it is 7...
- the string `help`. Since it is shorter than `version`, it is not covered by code
- The names of the selected subcommand flags
- The names of the global flags

Reply:
I am stupid for missing the `maxLength` being passed in again.  :-D